computer_vision/pooling_functions.py

```diff
@@ -1,11 +1,16 @@
-# Source : https://computersciencewiki.org/index.php/Max-pooling_/_Pooling
-# Importing the libraries
+"""
+Sources :
+ Max Pooling: https://computersciencewiki.org/index.php/Max-pooling_/_Pooling
+ Average Pooling: https://computersciencewiki.org/index.php/Average_pooling
+ Global Average Pooling: https://paperswithcode.com/method/global-average-pooling
+"""
+
 import numpy as np
 from PIL import Image
 
 
-# Maxpooling Function
-def maxpooling(arr: np.ndarray, size: int, stride: int) -> np.ndarray:
+# Max pooling Function
+def max_pooling(arr: np.ndarray, size: int, stride: int) -> np.ndarray:
     """
     This function is used to perform maxpooling on the input array of 2D matrix(image)
```

Comment:
```suggestion
def max_pooling(arr: np.ndarray, size: int, stride: int) -> np.ndarray:
    """
    Performs max pooling on the input image matrix
```
The comment at the top of the function is unnecessary because we already have the docstring to explain what the algorithm does